pkg/k8s/apis/cilium.io/register.go

```diff
@@ -15,5 +15,5 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.29.0"
+	CustomResourceDefinitionSchemaVersion = "1.29.1"
```

Comment:
Why does this change require a CRD version bump?

Reply:
We are modifying the schema - removing field in CiliumBGPFamily. I believe on upgrade if schema version is not modified, CRDs will not be updated. ( I am going by the comment at line 17 )